website/api/blockchain-integration/contracts/verifier.md

```diff
@@ -43,18 +43,19 @@ You can choose to use this contract or deploy your own.
 | ---------------------------------------------------------------- | -------------------------- | --------- | ---------------------------------------------------------------------- |
 | [RiscZeroGroth16Verifier.sol][RiscZeroGroth16Verifier.sol@v0.9]  | [v0.9.0][release-v0.9.0]   | [Sepolia] | [0xe57255C10291771A2E14f7eb257555AE82D78302][sepolia-verifier-v0.9.0]  |
 | [RiscZeroGroth16Verifier.sol][RiscZeroGroth16Verifier.sol@v0.10] | [v0.10.0][release-v0.10.0] | [Sepolia] | [0x4e9559f1fdf82dd1babdd42b4550ffdb1a1c59de][sepolia-verifier-v0.10.0] |
+| [RiscZeroGroth16Verifier.sol][RiscZeroGroth16Verifier.sol@v1.0]  | [v1.0.0][release-v1.0.0]   | [Sepolia] | [0x36be51af39a2d430368ffee8c664c46d2298083d][sepolia-verifier-v1.0.0] |
 
 [EvenNumber.sol@v0.10]: https://github.com/risc0/bonsai-foundry-template/blob/release-0.10/contracts/EvenNumber.sol
```

Comment:
This link to 0.10 could use a refresh -- not sure what the best way to manage maintenance is on these. 

I think when it comes to cutting new zkVM versions we have some systems in place so that links get updated during the release process, but I don't think we have those systems set up yet for risc0-ethereum versioning.